src/core/Microsoft.PowerToys.Settings.UI/Views/PowerRenamePage.xaml

```diff
@@ -73,14 +75,13 @@
             <CustomControls:GroupTitleTextBlock x:Uid="PowerRename_AutoCompleteHeader"
                                                 IsActive="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}"/>
             
-            <ToggleSwitch x:Uid="PowerRename_Toggle_AutoComplete"
+            <CheckBox x:Uid="PowerRename_Toggle_AutoComplete"
```

Comment:
Any motivation for the switch?

Reply:
A ToggleSwitch should be used for 'immediate' UI changes only. It also saves some space and makes it a bit more friendly on the eyes, but that's just a personal preference ;).

More info on toggeswitch vs. checkbox.
https://www.nngroup.com/articles/toggle-switch-guidelines/

Reply:
Cool website. Very informative!!!